app/services/team_service.rb

```diff
@@ -12,12 +12,11 @@ def invite(email, name)
     response = slack_client.admin_invite(email: email, first_name: name.to_s.scan(/^\S+/).first).symbolize_keys
     if response[:ok]
       ActivityWorker.perform_async @team.space_id, I18n.t('team_service.invited', name: name, email: email)
-      Rails.logger.info "#{@team.space.subdomain}: invited user #{email}/#{name}"
```

Comment:
would be nice to keep this with member id so if someone reports not getting an invite we can see whether it was us.